comm/src/test/scala/coop/rchain/p2p/URIParseSpec.scala

```diff
@@ -8,9 +8,11 @@ class URIParseSpec extends FlatSpec with Matchers {
     Left(ParseError(s"bad address: $s"))
 
   "A well formed rnode URI" should "parse into a PeerNode" in {
-    val uri = "rnode://abcde@localhost:12345"
+    val uri = "rnode://abcdef@localhost:12345"
     NetworkAddress.parse(uri) should be(
-      Right(PeerNode(NodeIdentifier("abcde".getBytes), Endpoint("localhost", 12345, 12345))))
+      Right(
+        PeerNode(NodeIdentifier("abcdef".sliding(2, 2).toArray.map(Integer.parseInt(_, 16).toByte)),
```

Comment:
testing assertion uses coded that is being tested, better: inline the result, potentially add comment

Reply:
good idea